toolbox-webseite.lektorproject

```diff
@@ -27,4 +27,5 @@ url_prefix = /sxu/
 locale = de
 
 [packages]
-lektor-webpack-support = 0.5
+lektor-jsminify = 1.0
+lektor-scsscompile = 1.0
```

Comment:
Uhh, was ist das?

Reply:
zwei von mir geschriebene Plugins für Lektor. Das eine minifiziert unseren javascript stuff und legt ihn in den assets ab, das andere compiliert sass. Beide finden sich auf pypi und haben config files in config/